README.md

```diff
@@ -73,13 +73,12 @@ necessary packages. You may want to run it as root.
 
 ```shell
 git clone https://github.com/rui314/mold.git
-mkdir mold/build
-cd mold/build
+cd mold
 git checkout v1.10.1
-../install-build-deps.sh
-cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=c++ ..
-cmake --build . -j $(nproc)
-sudo cmake --install .
+# Install required build dependencies
+./install-build-deps.sh
+# Build then install mold 
+./build-and-install
```

Comment:
It's named build-and-install.sh

Reply:
it explains the step actually, the script builds mold and then installs it

Reply:
ah lol got it sorry